src/main/java/org/javarosa/core/model/actions/SetValueAction.java

```diff
@@ -113,18 +94,14 @@ public TreeReference processAction(FormDef model, TreeReference contextRef) {
             //an unbound template, so see if such a reference could exist. Unfortunately this
             //won't be included in the above walk if the template is nested, since only the
             //top level template retains its subelement templates
-            if(model.getMainInstance().hasTemplatePath(target)) {
+            if (model.getMainInstance().hasTemplatePath(target)) {
```

Comment:
Same as above -- I tried to come up with a form def that would lead to a null resolved reference and couldn't do it but feel it's safer to leave the case in for now.